### ISSUE TITLE
Spinner inside modal dialog

### ISSUE THREAD

**Issue by ice-blaze** [oblique-bit/oblique#60]
The spinner seems not to work correctly inside a dialog. Inside Dialog component I tried to start the spinner through `ngOnInit` and `ngAfterViewInit`, but none is working. The spinner just appears a fraction of second. I removed `ObHttpApiInterceptor` but same behaviour. The only workaround was to add this line `<ng-template cdkConnectedOverlay [cdkConnectedOverlayOpen]="true"></ng-template>` just after the spinner and it seems to create an error in the console but then the spinner stays visible and can be controlled. 

I reproduce the bug with a clean angular project with oblique and added a button that open a dialog 

```html
<button (click)="startModal()">test</button>
```

```typescript
	public startModal(): void {
		this.dialog.open(DialogTestComponent);
	}
```

```typescript
export class DialogTestComponent {
	public spinnerChannel = 'spinner channel';
	constructor(private readonly spinnerService: ObSpinnerService) {}

	public ngAfterViewInit(): void {
		this.spinnerService.activate(this.spinnerChannel);
	}
}
```

```html
test<br />
test<br />
<ob-spinner [channel]="spinnerChannel" id="modalSpinner"></ob-spinner>
<!-- <ng-template cdkConnectedOverlay [cdkConnectedOverlayOpen]="true"></ng-template> -->
```

**Comment by gillerr** [comment]
The spinner did appear for a fraction a a second because of a bug in its animation. It did so even if not activated.

In this case, the spinner activation has to be done in `ngAfterViewInit``  otherwise the component that consume the event is not yet bound to the DOM. This do trigger an `ExpressionChangedAfterItHasBeenCheckedError` that can be avoided by wrapping the activation in a `setTimeout` call.

Both bugs have been fixed in the upcoming 8.2.0 version.

**Comment by gillerr** [comment]
Within a modal, a spinner has to be activated within `ngAfterViewInit ` . The spinner is now shown as expected and there's no ExpressionChangedAfterItHasBeenCheckedError `.
